### DISCUSSION TITLE
怎么直接发送一张图片？

### DISCUSSION THREAD

**Discussion by piwawa** [lobehub/lobehub#2457] [General Question]
调用 GPT4v 的时候，已经设置好了 system prompt，只需要发送图片即可，但是点发送或者回车都不能发送，必须输入文本一起发送。

![测试](https://github.com/lobehub/lobe-chat/assets/46926496/329966cb-0b6e-4560-ab12-653313632ca6)

**Comment by arvinxx** [comment]
这个我修一下，理论上应该要支持直接发送有图片的

**Comment by arvinxx** [comment]
已在修复 v1.0.14 修复
